private-templates-service/client/src/components/NavBar/NavBar.tsx

```diff
@@ -71,7 +90,7 @@ const NavBar = (props: NavBarProps) => {
         <Banner>
           <MobileBanner>
             <StyledLogo src={Logo} />
-            <Styledh1>{props.template ? props.template.name : props.currentPageTitle}</Styledh1>
+            <Styledh1>{props.templateName ? props.templateName : props.currentPageTitle}</Styledh1>
```

Comment:
I think we should try to take out references to `templateName` and `templateJSON` from the redux store and access them from the `template` object in the store so that there's one true source of template info. What do you think?

Reply:
done